scripts/gulp/plugins/util/tsLanguageService.ts

```diff
@@ -0,0 +1,82 @@
+import * as fs from 'fs';
+import path from 'path';
+import * as ts from 'typescript';
+
+export type TSFile = {
+  text?: string;
+  version: number;
+};
+
+function getTSConfigFile(tsconfigPath: string): ts.ParsedCommandLine {
```

Comment:
I'm curious, what's the reasoning for adding this more complicated logic rather than just parsing the tsconfig like in react-conformance?

Reply:
It works with `watch` without issues as files can be added/changed on the fly. Is it possible in your implementation?

Reply:
Ah okay, hadn't realized that was the purpose. Might be good to add a comment about that somewhere.

Reply:
Good point 👍 I will add comments tomorrow and will fix remaining things

Reply:
Done 👍